_partials/developer-materials-results.html.slim

```diff
@@ -41,7 +41,7 @@ div
             | Product
           td.skill Skill Level
       tbody
-        tr.material(ng-repeat="item in data.displayedMaterials" class="get-started-placeholder-{{item.fields.sys_type}}")
+        tr.material(ng-repeat="item in data.displayedMaterials | middlewareOnly" class="get-started-placeholder-{{item.fields.sys_type}}")
```

Comment:
@lukas-vlcek Can you provide an alternative to the following qury that filters out results related to a list of projects (e.g. 'rhel' and 'atomic')

http://dcp.jboss.org/v1/rest/search?content_provider=jboss-developer&content_provider=rht&field=sys_author&field=target_product&field=contributors&field=duration&field=github_repo_url&field=level&field=sys_contributors&field=sys_created&field=sys_description&field=sys_title&field=sys_tags&field=sys_url_view&field=thumbnail&field=sys_type&field=sys_rating_num&field=sys_rating_avg&field=experimental&query=sys_type:(video)&size=500